test/tests/functional/pbs_hook_set_jobenv.py

```diff
@@ -0,0 +1,959 @@
+                         " == " + self.env_nohook['TEST_DQUOTE6'].rstrip('\n'))
+        self.assertEqual(os.environ['TEST_SQUOTE'],
+                         self.env_nohook['TEST_SQUOTE'].rstrip('\n'))
+        self.logger.info("TEST_SQUOTE - " + os.environ['TEST_SQUOTE'] +
+                         " == " + self.env_nohook['TEST_SQUOTE'].rstrip('\n'))
+        self.assertEqual(os.environ['TEST_SQUOTE2'],
+                         self.env_nohook['TEST_SQUOTE2'].rstrip('\n'))
+        self.logger.info("TEST_SQUOTE2 - " + os.environ['TEST_SQUOTE2'] +
+                         " == " + self.env_nohook['TEST_SQUOTE2'].rstrip('\n'))
+        self.assertEqual(os.environ['TEST_SQUOTE3'],
+                         self.env_nohook['TEST_SQUOTE3'].rstrip('\n'))
+        self.logger.info("TEST_SQUOTE3 - " + os.environ['TEST_SQUOTE3'] +
+                         " == " + self.env_nohook['TEST_SQUOTE3'].rstrip('\n'))
+        self.assertEqual(os.environ['TEST_SQUOTE4'],
+                         self.env_nohook['TEST_SQUOTE4'].rstrip('\n'))
+        self.logger.info("TEST_SQUOTE4 - " + os.environ['TEST_SQUOTE4'] +
+                         " == " + self.env_nohook['TEST_SQUOTE4'].rstrip('\n'))
+        self.assertEqual(os.environ['TEST_SQUOTE5'],
+                         self.env_nohook['TEST_SQUOTE5'].rstrip('\n'))
+        self.logger.info("TEST_SQUOTE5 - " + os.environ['TEST_SQUOTE5'] +
+                         " == " + self.env_nohook['TEST_SQUOTE5'].rstrip('\n'))
+        self.assertEqual(os.environ['TEST_SQUOTE6'],
+                         self.env_nohook['TEST_SQUOTE6'].rstrip('\n'))
+        self.logger.info("TEST_SQUOTE6 - " + os.environ['TEST_SQUOTE6'] +
+                         " == " + self.env_nohook['TEST_SQUOTE6'].rstrip('\n'))
+        self.assertEqual(os.environ['BROL'],
+                         self.env_nohook['BROL'].rstrip('\n'))
+        self.logger.info("BROL - " + os.environ['BROL'] + " == " +
+                         self.env_nohook['BROL'].rstrip('\n'))
+        self.assertEqual(os.environ['BROL1'],
+                         self.env_nohook['BROL1'].rstrip('\n'))
+        self.logger.info("BROL - " + os.environ['BROL1'] + " == " +
+                         self.env_nohook['BROL1'].rstrip('\n'))
+
+        # match the values in qstat -f Variable_List
+        # Following is blocked on PTL bug PP-1008
+        # self.assertTrue("TEST_COMMA=1\,2\,3\,4" in job_var)
+        # self.assertTrue("TEST_SEMICOLON=\;" in job_var)
+        # self.assertTrue("TEST_COLON=:" in job_var)
+        # self.assertTrue("TEST_DQUOTE=\"" in job_var)
+        # self.assertTrue("TEST_SQUOTE=\'" in job_var)
+        # self.assertTrue("TEST_BACKSLASH=\\" in job_var)
+        # self.assertTrue("BROL=hii\\\\\\haha" in job_var)
+        # self.assertTrue("TEST_ENCLOSED=\," in job_var)
+        # self.assertTrue("BROL1=hii\nhaa" in job_var)
+        # self.assertTrue("TEST_RETURN=3\,\n4\,\n5\," in job_var)
```

Comment:
Remove commented lines of code

Reply:
Same reasoning as with the previous reply, the lines will get un-commented once PP-1008 is fixed.